src/elm/Community.elm

```diff
@@ -405,12 +406,29 @@ encodeCreateCommunityData c =
         , ( "description", Encode.string c.description )
         , ( "inviter_reward", Eos.encodeAsset c.inviterReward )
         , ( "invited_reward", Eos.encodeAsset c.invitedReward )
+        , ( "min_balance", Eos.encodeAsset c.minBalance )
```

Comment:
yeah, this will not work bro, you see there are two calls related to the creation of a community: 

1. Creating a community with details like logo, name, and invitation stuff 
2. Creating a Cambiatus Token, that communities may use.

A Community is an abstraction that allows for some stuff like: creating sales, having a network of people, configure that network invitation rules, create objetives, validate claims etc.

A token is monetary abstraction, it includes: defining negative limits, configuring supply, issuing new tokens etc.

This call you are doing is related to [this action](https://github.com/cambiatus/contracts/blob/1931f370299bb3bda181bb45c979509991cb52bc/community/community.hpp#L165-L167), and as you can see there is no `min_balance`, and that's because this is not a community rule, but rather a token rule. 

to create a token you can [call this action](https://github.com/cambiatus/contracts/blob/1931f370299bb3bda181bb45c979509991cb52bc/token/token.hpp#L50) and this one has the min balance. 

this is good! this means you are starting to interact with the blockchain and there a few ways to discover how it works. You can use the [Bloks block explorer](https://local.bloks.io/account/cambiatus.cm?nodeUrl=http%3A%2F%2Fstaging.cambiatus.io&coreSymbol=SYS&systemDomain=eosio&loadContract=true&tab=Actions&account=cambiatus.cm&scope=cambiatus.cm&limit=100&action=create) to explore what the contract can do and how you can call the actions or query its tables. 

we use two contracts on Cambiatus: `cambiatus.cm` for the community stuff and `cambiatus.tk` for the token stuff